pkg/id/id.go

```diff
@@ -61,3 +57,15 @@ func IsValid(s string) bool {
 func (id *ID) Time() time.Time {
 	return ulid.Time(id.value.Time())
 }
+
+func (id *ID) String() string {
+	return id.value.String()
+}
+
+// Must is a convenience function that is useful for tests and when starting up.
+func Must(id *ID, err error) *ID {
```

Comment:
@elbuo8 I know you don't like `Must` functions because they panic, but they are primarily used as helpers in tests. This is a really common pattern all over the Go community and even within the Go standard library. I think its nice to have for our tests, so I personally like the changes Craig has made here. I know you have asked me to change these kind of patterns in the past though, so I thought I would share my opinion further here.

Examples:
https://pkg.go.dev/net/netip#MustParseAddr
https://pkg.go.dev/html/template#Must
https://pkg.go.dev/regexp#MustCompile

https://pkg.go.dev/go.uber.org/zap#Must
https://pkg.go.dev/github.com/google/uuid#Must
https://pkg.go.dev/github.com/aws/aws-sdk-go/aws/session#Must